source/multiple-mongosyncs.txt

```diff
@@ -0,0 +1,255 @@
+         :copyable: false
+         :emphasize-lines: 3,9
+
+           shards: [
+             {
+               _id: 'shard01',
+               host: 'shard01/localhost:27501,localhost:27502,localhost:27503',
+               state: 1,
+               topologyTime: Timestamp({ t: 1656612236, i: 2 })
+             },
+             {
+               _id: 'shard02',
+               host: 'shard02/localhost:27504,localhost:27505,localhost:27506',
+               state: 1,
+               topologyTime: Timestamp({ t: 1656612240, i: 4 })
+             }
+           ]
+
+   .. step:: 3. Connect the ``mongosync`` instances
+
+      .. _c2c-shard-config-connect:
+
+      These instructions use a generic connection string. To modify the
+      connection string for your cluster architecture, refer to the
+      architecture specific :ref:`connection details <c2c-connecting>`.
+
+      .. tip::
+       
+         A single host server can run multiple ``mongosync`` instances. To
+         improve performance, run ``mongosync`` on multiple host servers.
+
+      Each ``mongosync`` instance:
+
+      - Connects to the same ``mongos`` instance in the source cluster.
+      - Connects to the same ``mongos`` instance in the destination
+        cluster.
+      - Connects to a single shard in the source cluster.
+      - Specifies a unique port to use for during synchronization.
+        Consider designating a range of ports to simplify scripting
+        {+c2c-product-name+} operations.
+
+      Run the first ``mongosync`` instance:
+
+      .. code-block:: javascript
+
+         mongosync \
+           --cluster0 "mongodb://user:password@cluster0host:27500" \
+           --cluster1 "mongodb://user:password@cluster1host:27500" \
+           --id shard01 --port 27601
+
+      Run a new ``mongosync`` instance for each shard in the source cluster.
+      Edit the ``--id`` and ``--port`` fields for each additional
+      ``mongosync`` instance.
+
+      .. code-block:: javascript
+        :emphasize-lines: 4
+
+         mongosync \
+           --cluster0 "mongodb://user:password@cluster0host:27500" \
+           --cluster1 "mongodb://user:password@cluster1host:27500" \
+           --id shard02 --port 27602
+
+.. _c2c-sharded-start:
+
+Start Multiple ``mongosync`` Instances
+--------------------------------------
+
+Use ``curl`` or another HTTP client to issue the :ref:`start
+<c2c-api-start>` command to each of the ``mongosync`` instances.
+
+.. code-block:: shell
+
+   curl monogsync01Host:27601/api/v1/start -XPOST --data \
+        '{ "source": "cluster0", "destination": "cluster1", \
+           "reversible": false, "enableUserWriteBlocking": false }'
+
+   curl monogsync02Host:27602/api/v1/start -XPOST --data \
+        '{ "source": "cluster0", "destination": "cluster1", \
+           "reversible": false, "enableUserWriteBlocking": false }'
+
+The ``start`` command options must be the same for all of the ``mongosync``
+instances.
+
+.. _c2c-sharded-progress:
```

Comment:
I had a look at the API docs, there isn't much duplication of their content. 

The 'this works on one shard, repeat on the others' part does get repeated in various forms. We could move it to the beginning of the section, but since these aren't steps we don't know how a reader arrives at a given procedure. They could easily miss a note that's farther up the page.

We could remove the example like this:


Check Progress
--------------

The ``mongosync`` instances do not aggregate progress information
across shards. To review synchronization progress for a particular
shard, use ``curl`` or another HTTP client to issue the
:ref:`progress <c2c-api-progress>` command to the ``mongosync``
instance synchronizing that shard. To check progress on other shards repeat
the API call to each ``mongosync`` instance.

But then we force the reader to jump to another page. Leaving the example in gives the user the basic code they need.